sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_connection_async.py

```diff
@@ -236,7 +236,7 @@ async def _read_frame(self, wait: Union[bool, int, float] = True, **kwargs) -> b
         """
         timeout: Optional[Union[int, float]] = None
         if wait is False:
-            timeout = 1  # TODO: What should this default be?
+            timeout = .2
```

Comment:
Weren't we going to move this to a constant?

Reply:
yes!